config/kafka-instance-types-configuration.yaml

```diff
@@ -8,13 +8,18 @@
 #
 # The following properties must be defined for each size (all values must be larger than '0'):
 #   - id: The size identifier. Each size id should be unique.
+#   - display_name: human readable value of the instance size
 #   - ingressThroughputPerSec: Maximum limit for ingress throughput per second (MB/s).
 #   - egressThroughputPerSec: Maximum limit for egress throughput per second (MB/s).
 #   - totalMaxConnections: Maximum client connections allowed.
 #   - maxConnectionAttemptsPerSec: Maximum client connection attempts per second.
 #   - maxDataRetentionSize: Maximum storage (GB).
 #   - maxDataRetentionPeriod: Maximum duration(ISO8601) for retaining data.
 #   - maxPartitions: Maximum partitions for all topics combined.
+#   - maxMessageSize: Maximum message size (MB)
+#   - minInSyncReplicas: Minimum number of in sync replicas
+#   - replicationFactor: Replication factor
+#   - supportedAZModes: a list of the supported AZ modes. The possible values are "single", "multi"
```

Comment:
Let's also add these parameters in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/212e5710729dd534b87f7fc98da0aab85851a03b/templates/service-template.yml#L163

and if possible and applicable create an issue/PR in kas-installer or just let people know of the change in parameters or addition of new parameters. 
/cc @MikeEdgar @agullon @daveconde FYI, once these PR is in, there will be extra parameters that will need to be configured for the `SUPPORTED_INSTANCE_TYPES` parameter

Reply:
Parameters updated

Reply:
Thanks. There are some people / pipeline that have this deployment parameter overridden. 
When this PR is merged, let's give a heads up in kas-installer chat with the fields to add to avoid surprise failures in their setup.